dashboard/src/pages/Dashboard/Dataset/DatasetSettingsPage.tsx

```diff
@@ -42,6 +42,7 @@ export const defaultServerEnvsConfiguration: ServerEnvsConfiguration = {
   COLLISIONS_ENABLED: false,
   FULLTEXT_ENABLED: true,
   QDRANT_COLLECTION_NAME: null,
+  EMBEDDING_QUERY_PREFIX: "",
```

Comment:
Shouldn't set this to `Search for: `?

Reply:
Shouldn't matter bc the server has it set as a default

Reply:
Wait no, we should set `Search For: ` only if they choose the jina embedding model. If they use the openai model it could be messed up

Reply:
But if you set it to empty it won't use the default right?

Reply:
If its empty, it should just be empty by default. The default ServerSetting should be empty

Reply:
> Wait no, we should set `Search For: ` only if they choose the jina embedding model. If they use the openai model it could be messed up

But then we need to backfill for other dataset settings that use Jina

Reply:
Not really. Just any new ones. this query prefix only gets applied at search time right?